src/peakPicking/peakPicking.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 /**
- * Implementation of the peak pickig method described by Cobas in:
+ * Implementation of the peak picking method described by Cobas in:
  * A new approach to improving automated analysis of proton NMR spectra
  * through Global Spectral Deconvolution (GSD)
  * http://www.spectroscopyeurope.com/images/stories/ColumnPDFs/TD_23_1.pdf
+ * @param {Object} spectrum - SD instance
+ * @param {Object} peakList - nmr signals
+ * @param {Object} options - options object with some parameter for GSD, detectSignal functions.
+ * @param {number} [options.nH] - Number of hydrogens or some number to normalize the integral data.
+ * @param {number} [options.integralFn] - option to chose between approx area with gaussian function or sum of the points of given range
```

Comment:
I'm not sure what number I should put? Maybe a string would be more intuitive. The list of possible values should be in the documentation.
If there is a default value, put it after the parameter name like this: `[options.integralFn='defaultvalue']`